homeassistant/components/intellifire/strings.json

```
@@ -5,23 +5,36 @@
       "manual_device_entry": {
         "description": "Local Configuration",
         "data": {
-          "host": "[%key:common::config_flow::data::host%]"
+          "host": "[%key:common::config_flow::data::host%] (IP Address)"
+        }
+      },
+      "api_config": {
+        "title": "IntelliFire - API Configuration",
+        "description": "IntelliFire will need to reach out to [iftapi.net](https://iftapi.net/webaccess/login.html) in order to obtain an API key. Once it has obtained this API key, the rest of its interactions will occur completely within the local network. If the API key were to expire it would again need to reach out to https://iftapi.net/webaccess/login.html\n\nUsername and Password are the same information used in your IntelliFire Android/iOS application. ",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%] (email)",
+          "password": "[%key:common::config_flow::data::password%]"
         }
       },
       "dhcp_confirm": {
         "description": "Do you want to setup {host}\nSerial: {serial}?"
       },
       "pick_device": {
+        "title": "Device Selection",
+        "description": "The following IntelliFire devices were discovered. Please select which you wish to configure.",
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
         }
       }
     },
     "error": {
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+      "cannot_connect": "Could not connect to a fireplace endpoint at url: http://{host}/poll\nVerify IP address and try again",
```

We don't need to expose such internal info as URLs. I think that this can be the old error.
```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
```